examples/kubernetes/proxy_with_sa_key.yaml

```diff
@@ -57,6 +57,20 @@ spec:
           mountPath: /secrets/
           readOnly: true
           # [END cloud_sql_proxy_k8s_volume_mount]
+        # Resource configuration depends on an application's requirements. The
+        # number of connections is correlated to memory use. The amount of IO
+        # across each connection is correlated to CPU use. Users should adjust
+        # the following values based on what their application needs.
+        # For details, see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+        resources:
+          requests:
+            # More connections require more memory. Fewer connections require
```

Comment:
Generally we prefer active over passive: 

```
The proxy's memory use scales linearly with the number of active connections. Fewer open connections 
will use less memory. Adjust this value based on your applications's requirements. 
```

Reply:
Fixed.